scripts/validate_diff_accounts.sh

```
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# Pre-commit hook to validate account definitions
+# This script checks that all rudderAccountId values in destination configs
+# have corresponding account configuration files.
+
+# Get the list of changed files
+changed_files=$(git diff --cached --name-only)
+
+# Check if any destination config files have been modified
+dest_config_pattern="src/configurations/destinations/.*/db-config\.json"
```

Shellcheck (0.10.0)

[warning] dest_config_pattern appears unused. Verify use (or export if used externally).

(SC2034)